apps/user/src/services/result/queries.ts

```diff
@@ -0,0 +1,26 @@
+import { Storage } from '@/apis/storage/storage';
+import { KEY, TOKEN } from '@/constants/common/constant';
+import { useQuery } from '@tanstack/react-query';
+import { getFinalResult, getFirstResult } from './api';
+
+export const useFirstResultQuery = () => {
+    const { data, ...restQuery } = useQuery({
+        queryKey: [KEY.FIRST_RESULT] as const,
+        queryFn: () => getFirstResult(),
```

Comment:
```suggestion
        queryFn: getFirstResult,
```
얘도 이렇게 줄일 수 있을 것 같아요